plugins/Makefile

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Intel Corporation
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#GOFLAGS :=
+#GO := go $(GOFLAGS)
+
+TOP = ..
+include $(TOP)/build.mk
+
+FABRIC_FPC_PLUGINS = $(FABRIC_PATH)/fpc_plugins
+
+ECC_VSCC_SRC = ecc-vscc/ecc_validation_plugin.go ecc-vscc/ecc_validation_logic.go
+ECC_VSCC_OUT = ecc-vscc.so
+
+ERCC_VSCC_SRC = ercc-vscc/ercc_validation_plugin.go ercc-vscc/ercc_validation_logic.go
+ERCC_VSCC_OUT = ercc-vscc.so
+
+ERCC_DECO_SRC = ercc-decorator/decoration.go
+ERCC_DECO_OUT = ercc-decorator.so
+
+build:
+	ln -sfn $$(/bin/pwd) $(FABRIC_FPC_PLUGINS); \
```

Comment:
recreating the symlink on each build feels a bit excessive but if it works ... :-)

Reply:
I happy for any suggestion here :D

Reply:
you could make FABRIC_FPC_PLUGINS a target (with the ln as the rule) and add dependency here?  (alternatively, you could test whether it already exists before creating `if [ -x .... `)?